contracts/plugins/assets/NonFiatCollateral.sol

```diff
@@ -125,11 +125,13 @@ contract NonFiatCollateral is Collateral {
 
     /// @return min {tok} The minimium trade size
     function minTradeSize() external view override returns (uint192 min) {
-        try uoaPerTargetFeed.price_(oracleTimeout) returns (uint192 p) {
+        try uoaPerTargetFeed.price_(oracleTimeout) returns (uint192 p1) {
             try chainlinkFeed.price_(oracleTimeout) returns (uint192 p2) {
                 // {UoA/tok} = {UoA/target} * {target/ref} * {ref/tok}
-                // p = p.mul(p2);
-                p = uint192((uint256(p) * p2) / FIX_ONE);
+
+                // In the next line we violate our standard practice and put D18 on a uint256
```

Comment:
Say not just that we do this, but why it's necessary; otherwise a future developer could sensibly come back through and "clean it up" in the future.

Reply:
Ditto on L153.

Reply:
Good point.